changelog.d/536.change.rst

```diff
@@ -0,0 +1,2 @@
+``is_callable()`` validator now raises a ``NotCallableError`` exception informing the value and type received.
```

Comment:
1. We use fully qualified names in the changelog (both is_callable and NotCallableError please)
2. We should stress that `NotCallableError` is a subclass of `TypeError`.